tools/js-optimizer.js

```diff
@@ -7572,7 +7572,8 @@ function emterpretify(ast) {
         func[3].push(srcToStat('sp = EMTSTACKTOP;'));
         var stackBytes = finalLocals*8;
         func[3].push(srcToStat('EMTSTACKTOP = EMTSTACKTOP + ' + stackBytes + ' | 0;'));
-        func[3].push(srcToStat('assert(((EMTSTACKTOP|0) <= (EMT_STACK_MAX|0))|0);'));
+        if (ASSERTIONS)
```

Comment:
Please add `{ }` around the if body (our convention is if it isn't on the same line, it needs `{ }`).

Otherwise, looks like there's a merge conflict from some recent landing. Otherwise looks good, thanks!

Reply:
Merge conflict is the change in emscripten.py, you now need something like
```
  if shared.Settings.EMTERPRETIFY:
```
(instead of settings['EMTERPRETIFY'])